docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -293,7 +293,7 @@ You can override the profile the executable is *built* with and *runs* with usin
 `./mvnw clean verify -Pnative -Dquarkus-profile=test`. This might come handy if there are test specific resources to be processed,
 such as importing test data into the database.
 
-```
+```properties
```

Comment:
This should be an Asciidoc source blocks with:
```
[source,properties]
----
```

Reply:
Maybe only change that if there's only one there. Otherwise, it's probably better to keep it that way and do it separately.

Reply:
Ok, I will remove it for the time being.